src/init.cpp

```diff
@@ -59,8 +59,7 @@
 #include <boost/algorithm/string/classification.hpp>
 #include <boost/algorithm/string/replace.hpp>
 #include <boost/algorithm/string/split.hpp>
-#include <boost/bind.hpp>
-#include <boost/thread.hpp>
+
```

Comment:
This file uses `boost::bind` and `boost::thread*`

Reply:
Looks like `boost/thread.h`  is included in `scheduler.h`

Reply:
Please revert, https://github.com/bitcoin/bitcoin/pull/14718#discussion_r233917853